plugins/openssl/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2025 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+# Component: openssl
+
+#############################################################
+# Description:
+# -----------
+# Cmake Input File for openssl
+#############################################################
+
+project(openssl)
+
+if (PLATFORM AND USE_OPENSSLV3)
+    find_package(OpenSSL REQUIRED)
+
+    set(
+        SRCS
+        openssl.hpp
+        openssl.cpp
+    )
+
+    include_directories(
+        ${HPCC_SOURCE_DIR}/system/include
+        ${HPCC_SOURCE_DIR}/rtl/eclrtl
+        ${HPCC_SOURCE_DIR}/rtl/include
+        ${HPCC_SOURCE_DIR}/common/deftype
+        ${HPCC_SOURCE_DIR}/system/jlib
+    )
+
+    add_definitions(-D_USRDLL -DECL_OPENSSL_EXPORTS)
+    HPCC_ADD_LIBRARY(openssl SHARED ${SRCS})
+
+    install(
+        TARGETS openssl
+        DESTINATION plugins CALC_DEPS
+    )
+    install(
+        FILES ${OPENSSL_LIBRARIES}
+        DESTINATION ${LIB_DIR}
+        COMPONENT Runtime
+    )
+
+    target_link_libraries(
+        openssl
+        eclrtl
+        jlib
+        OpenSSL::SSL
+        OpenSSL::Crypto
+    )
+endif()
```

Comment:
Looks like you need a final linefeed in the file.

Reply:
Added a linefeed.